pyseir/load_data.py

```diff
@@ -296,6 +298,22 @@ def load_county_metadata_by_fips(fips):
     return county_metadata_merged
 
 
+
+def get_all_fips_codes_for_a_state(state: str):
```

Comment:
This might be better in `utils.py`

Reply:
And you could also add an `@lru_cache()` decorator.

Reply:
the main reason i kept it in load_data, is because it's already colocated with load_county_metadata.  I figured some of these helper methods may make sense to re-arrange in a different pr.